docs/helpers/docs.ts

```diff
@@ -1,3 +1,15 @@
+/**
+ * Converts options array to a summary string.
+ *
+ * @param options Options array.
+ *
+ * @example
+ * optionsToSummary(['one', 'two'])
+ * // '"one" | "two"'
+ */
+export const optionsToSummary = (options: unknown) =>
+  (options as string[]).map((option) => `"${option}"`).join('|');
```

Comment:
Can't we type the input?

```suggestion
export const optionsToSummary = (options: readonly string[]) =>
  options.map((option) => `"${option}"`).join('|');
```